uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -709,6 +719,8 @@ define([
             }
 
             self.postContactSelection = false;
+
+            self._calculateOpeningAndClosingBalance();
```

Comment:
The position of calculating balance has changed, its not breaking anything right? (Double checking)